src/index.ts

```diff
@@ -344,6 +377,12 @@ function addNextIdMethods(chunks: Code[], config: Config): void {
   `);
 }
 
+function getScopedName(config: Config, typeName: string) {
```

Comment:
Hm, maybe call this `maybeImport`? Very small change, but I like the "maybe" to signal a "sometimes" behavior, i.e. it depends on the config setting.

Reply:
Updated!